recipes/msys/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "msys" %}
+{% set version = "1.7.258" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: git://github.com/rmcgibbo/msys.git
+    git_rev: "cf53228"
```

Comment:
This doesn't seem right: https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

Reply:
Yeah, as I said in the first update of the PR, this is still a work-in-progress. I've been iterating on the base recipe in order to debug build failures. When this is ready to go, I definitely won't be trying to build from git.

Reply:
I'd obviously like to merge the changes I've made to the project's SConscript upstream to https://github.com/DEShawResearch/msys before we merge this packaging PR.

Reply:
You could also apply those changes as a patch.

Reply:
The lead upstream developer is in the office two doors down from me, so I think we'll get it merged upstream shortly. :)

Do ya'll prefer building from a "release" tarball on github, or a tarball uploaded elsewhere?

Reply:
It is your choice. We have a bot which will issue PRs for version bumps if the tarballs are on most major platforms, including GitHub and PyPI. The bot will also try other places (eg. sourceforge), but doesn't always find the tarballs.

Reply:
Building against the tarball now.